pkgs/tools/misc/tensorman/default.nix

```diff
@@ -0,0 +1,26 @@
+#{ pkgconfig, stdenv, rustPlatform, rustc, cargo, docker, openssl, fetchzip }:
+
+rustPlatform.buildRustPackage rec {
+  name = "tensorman-${version}";
+  version = "0.1.0";
+
+ src = fetchzip{
```

Comment:
Please use ```fetchFromGitHub```.

Reply:
not possible as they do not track their releases on github yet